pymess/utils/html.py

```diff
@@ -0,0 +1,25 @@
+from typing import Optional, Sequence
+
+from bs4 import BeautifulSoup
+
+from django.core.exceptions import ValidationError
+from django.utils.translation import ugettext as _
+
+
+BANNED_TAGS = ('object', 'script', 'canvas', 'video', 'applet', 'embed', 'noscript', 'amp-iframe', 'application')
```

Comment:
dej do settings prosím

Reply:
jo to jsem chtěl udělat a zapomněl, díky